egs/multi_ja_en/ASR/prepare.sh

```
@@ -31,55 +33,54 @@ log "dl_dir: $dl_dir"
 if [ $stage -le 1 ] && [ $stop_stage -ge 1 ]; then
   log "Stage 1: Soft link fbank of musan"
-  mkdir -p data/fbank
   if [ -e ../../librispeech/ASR/data/fbank/.musan.done ]; then
-    cd data/fbank
-    ln -svf $(realpath ../../../../librispeech/ASR/data/fbank/musan_feats) .
-    ln -svf $(realpath ../../../../librispeech/ASR/data/fbank/musan_cuts.jsonl.gz) .
-    cd ../..
+    cd data/manifests
+    mkdir -p musan
+    cd musan
+    ln -svfr $(realpath ../../../../../librispeech/ASR/data/fbank/musan_feats) .
```

Shellcheck (0.10.0)

[warning] Quote this to prevent word splitting.

(SC2046)